Makefile

```diff
@@ -777,6 +795,14 @@ libs/network/js/network_js.c
  libs/network/wiznet/Ethernet/socket.c \
  libs/network/wiznet/W5500/w5500.c
  endif
+
```

Comment:
This might be left over from the other ESP8266 code that I pulled out in a commit a few days ago? Merging it anyway, but could be worth checking?